src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -332,6 +332,17 @@ std::string MessageHelper::CommonLanguageToString(
   return std::string();
 }
 
+std::string MessageHelper::MobileLanguageToString(
+    mobile_apis::Language::eType language) {
+  using namespace ns_smart_device_link::ns_smart_objects;
+  const char* str = 0;
+  if (EnumConversionHelper<mobile_apis::Language::eType>::EnumToCString(
+          language, &str)) {
+    return str ? str : "";
```

Comment:
@IGapchuk keep consistency at least in one method. On one place you return `""` as an empty string, In other `std::string()`. 
Also I would say that checking of pointer would be enough

Reply:
@LuxoftAKutsan Fixed in 048fe1415fce72016340c817c56367238ee052f6